system-addon/lib/ThemeFeed.jsm

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+"use strict";
+
+ChromeUtils.import("resource://gre/modules/Services.jsm");
+
+const {actionCreators: ac, actionTypes: at} = ChromeUtils.import("resource://activity-stream/common/Actions.jsm", {});
+
+const THEME_UPDATE_EVENT = "lightweight-theme-styling-update";
+
+this.ThemeFeed = class ThemeFeed {
+  init(LightweightThemeManager) {
+    Services.obs.addObserver(this, THEME_UPDATE_EVENT);
+
+    let temp = {LightweightThemeManager};
+    ChromeUtils.import("resource://gre/modules/LightweightThemeManager.jsm", temp);
```

Comment:
This seems odd… I guess for injecting custom LightweightThemeManager for testing? That's what the injector is for? Although in this case, we would just override the global if this was imported normally?

Reply:
I was assuming whoever did this knew what they were doing: https://searchfox.org/mozilla-central/source/toolkit/modules/LightweightThemeConsumer.jsm#29-30

But I could make it global if there is no reason to have done that.